flask_app/templates/index.html

```diff
@@ -15,6 +15,31 @@
 <div class="center">
   <h1>Success!</h1>
   <img src="http://rack.1.mshcdn.com/media/ZgkyMDEzLzA4LzA1Lzk3LzMwcm9jay40ZDgyNC5naWYKcAl0aHVtYgk4NTB4NTkwPgplCWpwZw/f1b84963/086/30-rock.jpg">
+  <div>Here is a <a href={{ url_for('secret') }}>secret page</a>. The default email is webmaster@weber.com and the password is 123456</div><br/>
```

Comment:
I wanted to make a demonstration of how the user should use authorization,  but I probably exaggerated. This example belongs to the documentation. I'll delete it from the template